src/value/ser.rs

```diff
@@ -247,23 +266,33 @@ impl serde::Serializer for Serializer {
     }
 }
 
+/// Not public API. Should be pub(crate).
```

Comment:
I don't think this and the following few similar comments are meaningful because these types are effectively already `pub(crate)`. They are not accessible from outside of serde_json. Please remove.

Reply:
Right. I can get rid of the `doc(hidden)` attributes too...